src/builtins/BuiltinTemporal.cpp

```diff
@@ -324,8 +324,11 @@ static Value builtinTemporalCalendarFrom(ExecutionState& state, Value thisValue,
 
 static Value builtinTemporalCalendarPrototypeId(ExecutionState& state, Value thisValue, size_t argc, Value* argv, Optional<Object*> newTarget)
 {
-    Value calendar = thisValue;
-    return calendar.asObject()->asTemporalCalendarObject()->getIdentifier();
+    if (!(thisValue.isObject() && thisValue.asObject()->isTemporalCalendarObject())) {
```

Comment:
I think there is a helper for this check.

Reply:
Yes but that function also checks if `argc` is not 0 and in this case `argc` will be 0.

Reply:
(IMHO)
Wrapping the result of  AND(`&&`) operation with a NOT(`!`) operation seems not quite intuitive.
What about changing it to OR(`||`)?